src/main/java/net/mcreator/ui/dialogs/tools/ToolPackMakerTool.java

```diff
@@ -197,6 +197,7 @@ static void addToolPackToWorkspace(MCreator mcreator, Workspace workspace, Strin
 		pickaxeTool.repairItems = Collections.singletonList(base);
 		pickaxeTool.creativeTab = new TabEntry(workspace, "TOOLS");
 		setParametersBasedOnFactorAndAddElement(mcreator, factor, pickaxeTool, folder);
+		pickaxeTool.attackSpeed = (double) Math.round(1.2f * factor);

```

Comment:
Isn't damage missing here?

Reply:
No, since by default it is set to 4
![image](https://github.com/MCreator/MCreator/assets/105946758/32e891ea-c96b-4b58-9a34-34b6dc35d7ab)